library/core/src/ffi/c_str.rs

```diff
@@ -322,8 +320,8 @@ impl CStr {
     /// assert_eq!(c_str.to_str().unwrap(), "AAAAAAAA");
     /// ```
     ///
-    #[unstable(feature = "cstr_from_bytes_until_nul", issue = "95027")]
-    #[rustc_const_unstable(feature = "cstr_from_bytes_until_nul", issue = "95027")]
+    #[stable(feature = "cstr_from_bytes_until_nul", since = "CURRENT_RUSTC_VERSION")]
+    #[rustc_const_unstable(feature = "const_cstr_from_bytes_until_nul", issue = "95027")]
```

Comment:
The constness could be stabilized at the same time, this function seems uncontroversial

Reply:
I initially did that, but it relies on `memchr` which isn’t yet const

Reply:
Ah, in that case, carry on. We'll stabilize the constness when the blockers are stable